src/rdb.c

```diff
@@ -3,41 +3,44 @@
 #include "rdb.h"
 
 dict *specDict_g_bkup;
-TrieMap *ScemaPrefixes_g_bkup;
+TrieMap *SchemaPrefixes_g_bkup;
```

Comment:
Is this renaming is part of this PR changes or is it unrelated? If its unrelated please avoid it next time as it make it much harder to concentrate on the relevant parts while review.